src/Windows/Markdown/Editor/Impl/Commands/MdCommandFactory.cs

```diff
@@ -15,16 +18,20 @@ namespace Microsoft.Markdown.Editor.Commands {
     [Export(typeof(ICommandFactory))]
     [ContentType(MdContentTypeDefinition.ContentType)]
     internal class MdCommandFactory : ICommandFactory {
+        private readonly IServiceContainer _services;
         private readonly IRInteractiveWorkflowProvider _workflowProvider;
 
         [ImportingConstructor]
-        public MdCommandFactory(IRInteractiveWorkflowProvider workflowProvider) {
-            _workflowProvider = workflowProvider;
+        public MdCommandFactory(ICoreShell coreShell)
+        {
```

Comment:
Yeah, it has its own settings. Though with at least two R# users, I think we can add a R# settings file to the repo as we did for VS settings. I'll do it.